lib/browser/api/browser-window.js

```diff
@@ -103,6 +104,15 @@ BrowserWindow.prototype._init = function () {
     app.emit('browser-window-focus', event, this)
   })
 
+  // Exit HTML fullscreen when window leaves fullscreen
+  this.on('leave-full-screen', (event) => {
+    for (const webContents of getAllWebContents()) {
+      if (webContents.getOwnerBrowserWindow() === this) {
```

Comment:
We can use `this.webContents` here instead of searching for all WebContents.

Reply:
@zcbenz I know, but there might be nested webContents